example/src/main/java/com/example/MainActivity.java

```diff
@@ -31,7 +35,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         // initialize the Parsely tracker with your site id and the current Context
-        ParselyTracker.init("example.com", 30, this, true);
+        ParselyTracker.init(DEFAULT_SITE_ID, 30, this, true);
```

Comment:
I wonder if `ParselyTracker.init()` shouldn't also receive a `SiteIdSource` 🤔 
Are there reasons not to?

Reply:
Because we need *some* site id to be defined. If `ParselyTracker.init()` received `SiteIdSource`, then API consumer could pass `SiteIdSource.Default`, which would have to be supported and it's not a valid state.

Alternatively, we could make `ParselyTracker.init()` receive `SiteIdSource.Custom`, but I think it only shadows implementation and causes confusion as it can raise a question: "custom" to what? In `ParselyTracker.trackPageview` it's "custom" to the value provided in initialization, so it makes sense there.

Reply:
> Because we need _some_ site id to be defined. If `ParselyTracker.init()` received `SiteIdSource`, then API consumer could pass `SiteIdSource.Default`, which would have to be supported and it's not a valid state.
> 
> Alternatively, we could make `ParselyTracker.init()` receive `SiteIdSource.Custom`, but I think it only shadows implementation and causes confusion as it can raise a question: "custom" to what? In `ParselyTracker.trackPageview` it's "custom" to the value provided in initialization, so it makes sense there.

Got it! Makes sense 👍